lib/tasks/check_card_deadline.rake

```diff
@@ -0,0 +1,17 @@
+require 'date'
+namespace :check_card_deadline do
+  desc 'TODO'
```

Comment:
__[Goodcheck]__

Complete all work!

Based on the comments it looks like some work was left for later.
Are you sure you want to continue?


<!-- Comment by [SideCI](https://sideci.com) -->